otc-swap-predicate/src/main.sw

```diff
@@ -0,0 +1,63 @@
+predicate;
+
+use std::{
+    inputs::{
+        input_count,
+        input_owner,
+    },
+    outputs::{
+        Output,
+        output_amount,
+        output_pointer,
+        output_type,
+    },
+};
+
+// TO DO : Remove once __gtf getters implemented in std-lib
+const GTF_OUTPUT_COIN_TO = 0x202;
+const GTF_OUTPUT_COIN_ASSET_ID = 0x204;
+
+/// Order / OTC swap Predicate
+fn main() -> bool {
+    // Order conditions: These are set in Forc.toml
+    // The spending transaction must have an output that sends `ask_amount` of `ask_token` to `receiver`
+    // Conversion to ContractId and Address types will be unnecessary once
+    // https://github.com/FuelLabs/sway/issues/2647 is fixed
+    let ask_token: ContractId = ContractId {
+        value: ask_token_config,
+    };
```

Comment:
```suggestion
    let ask_token = ContractId {
        value: ask_token_config,
    };
```

No need to be explicit here since the name of the struct is right there

Reply:
resolved in [c0a5fc1](https://github.com/FuelLabs/sway-applications/pull/142/commits/c0a5fc1c00ce18d3fafca269cb4e0fac78b2837d)